systemvm/patches/debian/config/opt/cloud/bin/vr_cfg.sh

```diff
@@ -27,39 +27,29 @@ log_it() {
     echo "$(date) : $*" >> $log
 }
 
-while getopts 'c:' OPTION
-do
-  case $OPTION in
-      c) cfg="$OPTARG"
-          ;;
-  esac
-done
+while getopts 'c:' OPTION; do
+    case $OPTION in
+        c) cfg="$OPTARG" ;;
+esac; done
 
-while read line
-do
+export DEFER_CONFIG=YESPLZ
```

Comment:
Also, maybe I am missing something, but I don't ever see this being used.  Why is it being set and how is it used?